datahub-web-react/src/app/ingest/source/builder/DefineRecipeStep.tsx

```diff
@@ -68,14 +69,8 @@ export const DefineRecipeStep = ({ state, updateState, goTo, prev }: StepProps)
     }, [stagedRecipeYml, showLookerBanner]);
 
     const onClickNext = () => {
-        // Convert the recipe into it's json representation, and catch + report exceptions while we do it.
-        let recipeJson;
-        try {
-            recipeJson = yamlToJson(stagedRecipeYml);
-        } catch (e) {
-            message.warn('Found invalid YAML. Please check your recipe configuration.');
-            return;
-        }
+        const recipeJson = getRecipeJson(stagedRecipeYml);
```

Comment:
I think we had this because it would throw if the yaml was malformed

Reply:
Nvm i see that if !recipeJson, we don't allow next ... But does the user know this?